src/bot.js

```diff
@@ -37,7 +37,7 @@ module.exports = app => {
 
   app.on(['issue_comment.created', 'issue_comment.edited'], async context => {
     const message = context.payload.comment.body
-    if (message.match(/merge to stag((ing)|e)/i)) {
+    if (message.match(/^merge to stag((ing)|e)$/i)) {
```

Comment:
```suggestion
    if (message.match(/^@some-app-name\s.*merge to stag((ing)|e)/i)) {
```

...require bot to be addressed by name, like dependabot does

Reply:
I don't think you can mention a bot. The way that dependabot does this is that you @ mention `dependabot` which is their organization. I don't think we want to @cruglobal to trigger a merge.

See https://github.com/CruGlobal/missionhub-api/pull/1463#issuecomment-645583906 for example.

Reply:
It could be a fake mention, just so that "@pattern" text matches

Reply:
It could :)